### PR TITLE
[release-0.7] docs: fix operator deployment instructions

### PR DESCRIPTION
**Pull request by marquiz**
(cherry picked from commit f302962519058644dbc52c30d662f1a948c4a06d)

### CODE REVIEW

File: docs/get-started/deployment-and-usage.md [hunk 1]
```diff
@@ -36,24 +36,32 @@ is recommended to be done via
...
-apiVersion: nfd.kubernetes.io/v1alpha1
-kind: NodeFeatureDiscovery
-metadata:
-  name: my-nfd-deployment
-  namespace: nfd
-EOF
-```
+
+   ```bash
+   kubectl create -f https://operatorhub.io/install/stable/nfd-operator.yaml
```
Comment (ArangoGutierrez) [inline docs/get-started/deployment-and-usage.md:41]: stable 😁

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]

File: docs/get-started/deployment-and-usage.md (hunk 1 above)
Comment (marquiz) [reply docs/get-started/deployment-and-usage.md]: Yeah, fixed on the second cherry-picked commit

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Pull request merged**